### DISCUSSION TITLE
qodana-python-community image

### DISCUSSION THREAD

**Discussion by vonZeppelin** [JetBrains/Qodana#163] [Ideas]
Hello,

As there are IntelliJ IDEA Community Edition and `qodana-jvm-community` Qodana image, will `qodana-python-community` Qodana image be ever released (based on PyCharm Community?).

If not, will installation of the [Python Community Edition plugin](https://plugins.jetbrains.com/plugin/7322-python-community-edition) into the `qodana-jvm-community` Qodana image unlock Python code inspections?

Thank you!

**Comment by brichbash** [comment]
Hello,

We will start publishing `qodana-python-community` linter after release, and it will be based on PyCharm Community.

> will installation of the [Python Community Edition plugin](https://plugins.jetbrains.com/plugin/7322-python-community-edition) into the qodana-jvm-community Qodana image unlock Python code inspections?

It will work. You can check[ our documentation](https://www.jetbrains.com/help/qodana/2022.3/qodana-python-docker-techs.html#Manage+plugins) for more information on plugin installation.

[a comment by an automated account is not shown]
